Ratings_MVP/Ratings_MVP/Services/RealmService/RealmService.swift

```diff
@@ -0,0 +1,41 @@
+// RealmService.swift
+// Copyright © RoadMap. All rights reserved.
+
+import RealmSwift
+
+/// Realm service
+final class RealmService: RealmServiceProtocol {
+    // MARK: - Public Properties
+
+    static let deleteIfMigration = Realm.Configuration(deleteRealmIfMigrationNeeded: true)
+
+    // MARK: - Public Methods
+
+    static func save<T: Object>(
+        items: [T],
```

Comment:
```suggestion
        items: [T],
```
В целом, дженерик тут не обязателен, можешь просто массив объектов сохранять, у тебя все равно ниче не сохранится, если оно не конформист обжект